alembic (0.6.2)
anaconda-mode (0.1.1)
appdirs (1.4.0)
apt-xapian-index (0.45)
argh (0.26.2)
argparse (1.2.1)
autoflake (0.6.6)
backports.shutil-get-terminal-size (1.0.0)
baron (0.6.2)
blinker (1.3)
chardet (2.0.1)
click (6.6)
colorama (0.2.5)
cssmin (0.2.0)
decorator (4.0.9)
dominate (2.2.0)
enum34 (1.1.6)
Flask (0.11.1)
Flask-Admin (1.4.2)
Flask-Assets (0.12)
Flask-Bootstrap (3.3.5.7)
flask-csrf (0.9.2)
Flask-DebugToolbar (0.10.0)
flask-ext-migrate (0.1.0)
Flask-FlatPages (0.6)
Flask-Login (0.3.1)
Flask-Mail (0.9.1)
Flask-Migrate (1.1.0)
Flask-Moment (0.2.1)
Flask-Script (2.0.5)
Flask-SQLAlchemy (2.1)
Flask-WhooshAlchemyPlus (0.7.5)
Flask-WTF (0.9.4)
gevent (1.0)
github3.py (1.0.0a4)
greenlet (0.4.10)
gunicorn (19.6.0)
gyp (0.1)
html5lib (0.999)
httpie (0.9.3)
ipdb (0.10.0)
ipython (4.2.0)
ipython-genutils (0.1.0)
itsdangerous (0.24)
jedi (0.9.0)
jieba (0.38)
Jinja2 (2.7.1)
jsmin (2.2.1)
json-rpc (1.8.1)
Mako (0.9.1)
Markdown (2.6.7)
MarkupSafe (0.23)
meld3 (1.0.2)
ndg-httpsclient (0.4.2)
nose (1.3.7)
pathlib (1.0.1)
pathlib2 (2.1.0)
pathtools (0.1.2)
pexpect (4.0.1)
pickleshare (0.7.2)
pip (1.5.4)
psutil (4.3.1)
ptyprocess (0.5.1)
pyasn1 (0.1.9)
pycurl (7.19.3)
pyflakes (1.2.2)
Pygments (2.1.3)
pygraphviz (1.2)
pyOpenSSL (16.0.0)
pyScss (1.3.5)
Exception:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/usr/lib/python2.7/dist-packages/pip/commands/list.py", line 80, in run
    self.run_listing(options)
  File "/usr/lib/python2.7/dist-packages/pip/commands/list.py", line 142, in run_listing
    self.output_package_listing(installed_packages)
  File "/usr/lib/python2.7/dist-packages/pip/commands/list.py", line 151, in output_package_listing
    if dist_is_editable(dist):
  File "/usr/lib/python2.7/dist-packages/pip/util.py", line 366, in dist_is_editable
    req = FrozenRequirement.from_dist(dist, [])
  File "/usr/lib/python2.7/dist-packages/pip/__init__.py", line 286, in from_dist
    assert len(specs) == 1 and specs[0][0] == '=='
AssertionError

Storing debug log for failure in /root/.pip/pip.log
